cyences_app_for_splunk/default/data/ui/views/cs_cisco_meraki.xml

```diff
@@ -0,0 +1,111 @@
+        <search>
+          <query>`cs_cisco_meraki` sourcetype="meraki:organizationsecurity" | dedup eventType</query>
+          <earliest>$timeRange.earliest$</earliest>
+          <latest>$timeRange.latest$</latest>
+        </search>
+        <choice value="*">All</choice>
+        <default>*</default>
+        <fieldForLabel>eventType</fieldForLabel>
+        <fieldForValue>eventType</fieldForValue>
+      </input>
+      <table>
+        <search>
+          <query>`cs_cisco_meraki` sourcetype="meraki:organizationsecurity" eventType=$tkn_event_type|s$ | table _time, eventType, message, priority, srcIp, destIp, deviceMac, clientMac, protocol, blocked, classification</query>
+          <earliest>$timeRange.earliest$</earliest>
+          <latest>$timeRange.latest$</latest>
+          <sampleRatio>1</sampleRatio>
+        </search>
+        <option name="drilldown">none</option>
+        <option name="refresh.display">progressbar</option>
+      </table>
+    </panel>
+  </row>
+  <row>
+    <panel>
+      <title>Config Changes</title>
+      <input type="dropdown" token="tkn_admin_email">
+        <label>Admin</label>
+        <search>
+          <query>`cs_cisco_meraki` sourcetype="meraki:audit" | dedup adminEmail | eval adminLabel=adminName." (".adminEmail.")"</query>
+          <earliest>$timeRange.earliest$</earliest>
+          <latest>$timeRange.latest$</latest>
+        </search>
+        <choice value="*">All</choice>
+        <default>*</default>
+        <fieldForLabel>adminLabel</fieldForLabel>
+        <fieldForValue>adminEmail</fieldForValue>
+      </input>
+      <input type="dropdown" token="tkn_network">
+        <label>Network</label>
+        <choice value="*">All</choice>
+        <default>*</default>
+        <search>
+          <query>`cs_cisco_meraki` sourcetype="meraki:audit" | fillnull networkName value="-None-" | dedup networkName</query>
+          <earliest>$timeRange.earliest$</earliest>
+          <latest>$timeRange.latest$</latest>
+        </search>
+        <fieldForLabel>networkName</fieldForLabel>
+        <fieldForValue>networkName</fieldForValue>
+      </input>
+      <input type="dropdown" token="tkn_page">
+        <label>Configuration Page</label>
+        <choice value="*">All</choice>
+        <default>*</default>
+        <fieldForLabel>page</fieldForLabel>
+        <fieldForValue>page</fieldForValue>
+        <search>
+          <query>`cs_cisco_meraki` sourcetype="meraki:audit" | dedup page</query>
+          <earliest>$timeRange.earliest$</earliest>
+          <latest>$timeRange.latest$</latest>
+        </search>
+      </input>
+      <table>
+        <search>
+          <query>`cs_cisco_meraki` sourcetype="meraki:audit"  adminEmail=$tkn_admin_email|s$ page=$tkn_page|s$ | fillnull networkName value="-None-" | search networkName=$tkn_network|s$ | eval time=strftime(_time, "%F %T %Z") | table time, page, networkName, networkId, adminName, adminEmail, label, oldValue, newValue</query>
```

Comment:
Use | cs_human_readable_time_format(_time, event_time)  instead of | eval time=strftime(_time, "%F %T %Z")